lib/CMakeLists.txt

```diff
@@ -124,11 +124,16 @@ else()  # WIN32
 endif()  # WIN32
 
 # Internal flags for coverage builds:
+set(JPEGXL_COVERAGE_FLAGS)
```

Comment:
`JPEGXL_COVERAGE_FLAGS`

is this flag used anywhere?

Reply:
Not really. This seems to be a bug in clang - sometimes it picks wrong compiler-rt, when not asked explicitly.